src/bindings/c/src/ov_prepostprocess.cpp

```diff
@@ -524,3 +524,23 @@ ov_status_e ov_preprocess_prepostprocessor_build(const ov_preprocess_prepostproc
 
     return ov_status_e::OK;
 }
+
+ov_status_e ov_preprocess_prepostprocessor_pad(ov_preprocess_prepostprocessor_t* preprocess_input_process_steps,
```

Comment:
```suggestion
ov_status_e ov_preprocess_preprocess_steps_pad(ov_preprocess_prepostprocessor_t* preprocess_input_process_steps,
```

Move this function to line 208